environnement-risques-and-sante.csl

```diff
@@ -0,0 +1,130 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" default-locale="en-US">
+  <info>
+    <title>Environnement Risques et SantÃ©</title>
```

Comment:
since the journal uses & in its title, I'd use that, too. You'll have to write it as ```&amp;```